src/main/java/store/bookshop/model/Book.java

```diff
@@ -24,12 +26,17 @@ public class Book {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+    @NotNull
     @Column(name = "title", nullable = false)
     private String title;
+    @NotNull
     @Column(name = "author", nullable = false)
     private String author;
+    @NotNull
     @Column(name = "isbn", nullable = false, unique = true)
     private String isbn;
+    @NotNull
+    @Min(value = 0)
     @Column(name = "price", nullable = false)
```

Comment:
Lets validate more directly, using more valid annotation @isbn @min @notempty and others